scripts/cargo_metadata_dump.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from pathlib import Path
+import json
+import os
+import sys
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from utils import run_cmd
+
+
+def save_app_params(device: str, app_build_path: Path, json_path: Path) -> None:
+    metadata = run_cmd("cargo metadata --no-deps --format-version 1 --offline -q",
+                       cwd=app_build_path)
+    metadata = json.loads(metadata)
```

Comment:
From Python you could use `toml` directly to extract the necessary info from the `Cargo.toml` (avoids a conversion to json)

Reply:
As discussed in direct, I'm keeping the usage of cargo metadata as this do'nt cost much and might allow detect weird conflict case in toml file which could be handled differently by cargo and python toml lib.